docs/fleetautoscaler_spec.md

```diff
@@ -31,12 +49,78 @@ The `spec` field is the actual `FleetAutoscaler` specification and it is compose
 - `fleetName` is name of the fleet to attach to and control. Must be an existing `Fleet` in the same namespace
    as this `FleetAutoscaler`.
 - `policy` is the autoscaling policy
-  - `type` is type of the policy. For now, only "Buffer" is available
-  - `buffer` parameters of the buffer policy
+  - `type` is type of the policy. "Buffer" and "Webhook" are available
+  - `buffer` parameters of the buffer policy type
     - `bufferSize`  is the size of a buffer of "ready" game server instances
                     The FleetAutoscaler will scale the fleet up and down trying to maintain this buffer, 
                     as instances are being allocated or terminated
                     it can be specified either in absolute (i.e. 5) or percentage format (i.e. 5%)
     - `minReplicas` is the minimum fleet size to be set by this FleetAutoscaler. 
                     if not specified, the minimum fleet size will be bufferSize
-    - `maxReplicas` is the maximum fleet size that can be set by this FleetAutoscaler. Required. 
\ No newline at end of file
+    - `maxReplicas` is the maximum fleet size that can be set by this FleetAutoscaler. Required. 
+  - `webhook` parameters of the webhook policy type
+    - `service` parameters
+      - `name`  is the service name bound to Deployment of autoscaler webhook [(see example)](../examples/autoscaler-webhook/autoscaler-service.yaml)
```

Comment:
It seems that this is from a `WebhookClientConfig` we should give a link to that spec for more details.